ui/app/src/components/shared/RowanPrice.vue

```diff
@@ -1,26 +1,30 @@
 <script lang="ts">
 import { defineComponent } from "vue";
 
-export default defineComponent({
-  components: {},
-  async setup() {
-
-    function isNumeric(s: any) {
-      return (s - 0) == s && (''+s).trim().length > 0;
-    }
-
-    const data = await fetch("https://vtdbgplqd6.execute-api.us-west-2.amazonaws.com/default/tokenstats");
-    const json = await data.json();
-    const rowanPriceInUSDT = json.body ? json.body.rowanUSD : "";
-
-    let rowanUSD = "";
-    if (isNumeric(rowanPriceInUSDT)) {
-      rowanUSD = "ROWAN: $" + parseFloat(rowanPriceInUSDT).toPrecision(6);
-    }
 
+export default defineComponent({
+  data() {
     return {
-      rowanUSD
+      rowanUSD: ""
+  }},
+  methods: {
+    async pollData () {
+      function isNumeric(s: any) {
+        return (s - 0) == s && (''+s).trim().length > 0;
+      }
+      const data = await fetch("https://vtdbgplqd6.execute-api.us-west-2.amazonaws.com/default/tokenstats");
+      const json = await data.json();
+      const rowanPriceInUSDT = json.body ? json.body.rowanUSD : "";
+
+      console.log('rowanPriceInUSDT')
+      if (isNumeric(rowanPriceInUSDT)) {
+        this.rowanUSD = "ROWAN: $" + parseFloat(rowanPriceInUSDT).toPrecision(6);
+      }   
     }
+  },
+  created() {
```

Comment:
Should this clear the interval when the component is unmounted or does it not matter in this instance?

Reply:
It wouldn't matter in this instance as the component should never be destroyed. But it's probably good practice either way